cmd/minikube/cmd/start.go

```diff
@@ -88,6 +88,9 @@ type versionJSON struct {
 	Commit          string `json:"commit"`
 }
 
+// ErrKubernetesPatchNotFound is when a patch was not found for the given <major>.<minor> version
+var ErrKubernetesPatchNotFound = errors.New("Unable to detect the latest patch release for specified Kubernetes version")
+
 var (
```

Comment:
```suggestion
var (
	// ErrKubernetesPatchNotFound is when a patch was not found for the given <major>.<minor> version
	ErrKubernetesPatchNotFound = errors.New("Unable to detect the latest patch release for specified Kubernetes version")
```
nit: This could go in the vars below